### ISSUE TITLE
False Advertising

### ISSUE THREAD

**Issue by moritztim** [upscayl/upscayl.github.io#9]
I reverse searched the high res image and found a stock image that matches pixel by pixel and I refuse to believe that `high.jpg` was generated from `low.jpg`.
https://www.shutterstock.com/shutterstock/photos/2346215871/display_1500/stock-photo-adventure-amazing-landscape-nature-photography-2346215871.jpg
https://www.upscayl.org/high.jpg
https://www.upscayl.org/low.jpg

**Comment by NayamAmarshe** [comment]
Yes, that is true. It's only a comparison slider to showcase what the project is for.

I was thinking of updating the website to automatically loop through some examples. The one on the website is the same as what I made on Figma, so I kept it as is.

**Comment by moritztim** [comment]
I still don't understand why you don't just use an image that was upscaled with this tool.

**Comment by NayamAmarshe** [comment]
> I still don't understand why you don't just use an image that was upscaled with this tool.

That's what I wanted to do, a whole carousel of images upscaled with Upscayl but I'm only a person and I have to assign priorities to other things at the moment.

**Comment by moritztim** [comment]
That is fair but I think you should still simply replace the images that are currently used, until you get around to the carousel.

**Comment by NayamAmarshe** [comment]
To be honest, I think the current image is fine for now, it gets the point across.

I'll replace it later with some other examples when I get to the website updates :)

**Comment by MrPenguin07** [comment]
> I still don't understand why you don't just use an image that was upscaled with this tool.

Why don't McDonalds get their menu pictures from real burgers?
Same principle :)
FWIW I think the dramatic effect is eye catching, and is part of the advertising. I wouldn't expect the image used for icons/website would be a real example - unless it was specifically stated as such.

**Comment by moritztim** [comment]
Because McDonalds gets more money if they make fake pictures. And they explicitly state in their advertisements that this ain't what you get because it would be false advertisement if they didn't.
I would expect this to be real because it's an interactive slider and it lies below a header saying "From Science Fiction to Reality", together, this gives you very high expectations and I think a user who saw the website first, then tried the tool would be disappointed to see the actual result (just like I am whenever I decide to give McDonalds one more chance). I didn't see the website first and was amazed by the tool's capabilities because I didn't have unrealistic expectations.
The difference to burgers is that if McDonalds used actual products in their ads, you'd see the Burger King next door and go there instead. However, I haven't seen an alternative to this tool with the same kind of advertising.

**Comment by MrPenguin07** [comment]
> Because McDonalds gets more money if they make fake pictures. And they explicitly state in their advertisements that this ain't what you get because it would be false advertisement if they didn't. I would expect this to be real because it's an interactive slider and it lies below a header saying "From Science Fiction to Reality", together, this gives you very high expectations and I think a user who saw the website first, then tried the tool would be disappointed to see the actual result (just like I am whenever I decide to give McDonalds one more chance). I didn't see the website first and was amazed by the tool's capabilities because I didn't have unrealistic expectations. The difference to burgers is that if McDonalds used actual products in their ads, you'd see the Burger King next door and go there instead. However, I haven't seen an alternative to this tool with the same kind of advertising.

Well I guess the question becomes; how common are your expectations?
Why hasn't McD - or literally every restaurant on earth - started using realistic images of exactly what you'll get?

Anyway I do agree in principle about realism in society, but here I guess the point is to catch peoples short attention span  and have them understand what the program does in a nanosecond before even needing to read it's description....for that the dramaticized image does it's job well imo.

**Comment by moritztim** [comment]
I get where you're coming from but regarding the capabilities of a software, I just don't like being lied to.

Besides, the tool itself is impressive but all this does for me is making it less so by setting unrealistic expectations. Here's a quick example showing the actual capabilities, which is still impressive, just not as impressive as taking an actual high res picture:
![low](https://github.com/upscayl/upscayl.github.io/assets/90388353/4a6b2094-ec54-4ccd-a5cd-cf56245a40c8)
![low_upscayl_4x_ultrasharp](https://github.com/upscayl/upscayl.github.io/assets/90388353/c95eba0d-b173-40c5-92cc-70eccc9a8638)
I think with a bit of tweaking, a real example like this could go on the website
